### ISSUE TITLE
Datally

### ISSUE THREAD

**Issue by rogersachan** [codyogden/killedbygoogle#638]
**What's the Product's Name?**
Datally
**Describe the product in a single sentence.**
```
It’s a mobile data manager that will help you monitor, save, and gain control of your data. Through tracking and controlling app data usage, you save more data and get the most out of your phone.
```
**When was the product launched? Provide a specific date, if possible.**
June 2017 per https://thenextweb.com/apps/2017/11/30/googles-data-saver-app-for-android-gets-a-wi-fi-finder-and-a-new-name/ (was named "Triangle" then)
**When was the product discontinued? Provide a specific date, if possible.**
10/17/2019
**What type of product was it? App, Service, or Hardware?**
App

https://www.androidpolice.com/2019/10/16/datally-app-gone/

**Comment by mskadu** [comment]
Done. Pending merge by @codyogden

**Comment by mskadu** [comment]
Ok, NOW i am done. Waiting for @codyogden to verify and merge 👍

**Comment by codyogden** [comment]
Per my comment in #640 

https://github.com/codyogden/killedbygoogle/pull/640#pullrequestreview-303843993

**Comment by mskadu** [comment]
Hi @codyogden, sorry to have missed this. I have left a response on the PR. But in a nutshell even TNW are calling it dead 😇

https://thenextweb.com/google/2019/10/17/googles-datally-app-for-saving-on-mobile-data-is-dead/

**Comment by codyogden** [comment]
Resolved in #640
